cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/CxxCoverageSensor.java

```diff
@@ -36,33 +37,50 @@
 import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.batch.sensor.SensorDescriptor;
 import org.sonar.api.batch.sensor.coverage.NewCoverage;
+import org.sonar.api.config.Settings;
 import org.sonar.api.batch.sensor.coverage.CoverageType;
+import org.sonar.api.utils.Version;
 import org.sonar.api.utils.log.Logger;
 import org.sonar.api.utils.log.Loggers;
 import org.sonar.cxx.CxxLanguage;
 import org.sonar.cxx.sensors.utils.CxxReportSensor;
 import org.sonar.cxx.sensors.utils.CxxUtils;
+import org.sonar.cxx.sensors.utils.EmptyReportException;
 
 /**
  * {@inheritDoc}
  */
 public class CxxCoverageSensor extends CxxReportSensor {
   private static final Logger LOG = Loggers.get(CxxCoverageSensor.class);
+
+  // Configuration properties before SQ 6.2
+  @Deprecated
   public static final String REPORT_PATH_KEY = "coverage.reportPath";
+  @Deprecated
   public static final String IT_REPORT_PATH_KEY = "coverage.itReportPath";
+  @Deprecated
   public static final String OVERALL_REPORT_PATH_KEY = "coverage.overallReportPath";
+  @Deprecated
   public static final String FORCE_ZERO_COVERAGE_KEY = "coverage.forceZeroCoverage";
-  
+
+  // Configuration properties for SQ 6.2
+  public static final Version SQ_6_2 = Version.create(6, 2);
+  private boolean isSQ_6_2_or_newer;
+  public static final String REPORT_PATHS_KEY = "coverage.reportPaths";
```

Comment:
@Bertk we have now ReportPath, ReportPatths and your new ReportPaths. Please use only `ReportPath`. In a final step we should change everything from ReportPath => ReportPatths (but only after everything is working again). Here you see the different flavours :https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Supported-configuration-properties.